### PR TITLE
chore: wrap error unhandling example in assertRejects

### PR DESCRIPTION
**Pull request by ryanleecode**

### CODE REVIEW

File: examples/error_unhandling.ts [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+    .transfer({
+      value: 1_000_000_000_000_000_000_000_000_000_000_000_000n,
+      dest: bob.address,
+    })
+    .signed({ sender: alice })
+    .sent()
+    .dbgStatus()
+    .finalizedEvents()
+    .unhandleFailed()
+    .run(), "unhandleFailed")
```
Comment (tjjfvi) [inline examples/error_unhandling.ts:16]: I don't think the `"unhandleFailed"` here is right.

Comment (tjjfvi) [reply examples/error_unhandling.ts:16]:
```suggestion
    .run())
```

Comment (ryanleecode) [reply examples/error_unhandling.ts:16]: mmm gimme a suggestion. not sure how these should be worded. "should undandleFailed` ?

Comment (ryanleecode) [reply examples/error_unhandling.ts:16]: oh just remove it ok

Comment (tjjfvi) [reply examples/error_unhandling.ts:16]: Ohh, sorry, I misunderstood that parameter. In that case, perhaps `"Should fail due to `InsufficientBalance`"`?

**Review by tjjfvi — APPROVED** [review]

**Comment by ryanleecode** [comment]
@tjjfvi override merge plz

**Pull request merged**